README.md

```diff
@@ -1,68 +1,32 @@
+MinIO Operator brings native support for [MinIO](https://github.com/minio/minio), [Console](https://github.com/minio/console), and [KES](https://github.com/minio/kes) to Kubernetes.
 
-MinIO-Operator brings native MinIO, [Console](https://github.com/minio/console), and [KES](https://github.com/minio/kes) support to Kubernetes. MinIO-Operator currently supports following features:
+## Prerequisites
 
-| Feature                 | Reference Document |
-|-------------------------|--------------------|
-| Create and delete highly available distributed MinIO clusters  | [Create a MinIO Instance](https://github.com/minio/operator#create-a-minio-instance). |
-| TLS Configuration  | [TLS for MinIO Instance](https://github.com/minio/operator/blob/master/docs/tls.md). |
-| Expand an existing MinIO cluster | [Expand a MinIO Cluster](https://github.com/minio/operator/blob/master/docs/adding-zones.md). |
-| Use a custom template for hostname discovery | [Custom Hostname Discovery](https://github.com/minio/operator/blob/master/docs/custom-name-templates.md). |
-| Use PodSecurityPolicy for MinIO Pods | [Apply PodSecurityPolicy](https://github.com/minio/operator/blob/master/docs/pod-security-policy.md). |
-| Deploy Console with MinIO cluster  | [Deploy MinIO Instance with Console](https://github.com/minio/operator/blob/master/docs/console.md). |
-| Deploy KES with MinIO cluster  | [Deploy MinIO Instance with KES](https://github.com/minio/operator/blob/master/docs/kes.md). |
+- Kubernetes >= v1.17.0.
+- Create PVs using [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md).
+- Install `kubectl minio` plugin.
 
-## Getting Started
+## Steps to Run
 
-### Prerequisites
-
-- Kubernetes version v1.17.0 and above for compatibility. MinIO Operator uses `k8s/client-go` v0.18.0.
-- `kubectl` configured to refer to a Kubernetes cluster.
-- Create the required PVs using [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md).
-- Optional: `kustomize` installed as [explained here](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md#installation).
-
-### Create Operator Deployment
-
-To start MinIO-Operator with default configuration, use the `operator.yaml` file.
-
-```bash
-kubectl apply -f https://raw.githubusercontent.com/minio/operator/master/minio-operator.yaml
+```sh
+kubectl minio operator create
+kubectl create ns tenant1-ns
+kubectl create secret generic minio-secret --from-literal=accesskey=minio --from-literal=secretkey=minio123 --namespace tenant1-ns
+kubectl minio tenant create --name tenant1 --secret minio-secret --servers 4 --volumes 4 --capacity 1Ti --namespace tenant1-ns
 ```
```

Comment:
Should add connection information. 

"To connect to the created cluster, add the cluster to your ``mc`` configuration and perform operations on it:

```
mc alias set tenant1-ns https://<service hostname>:9000  minio minio123

mc admin info
```

Reply:
I'm assuming we auto-create the service for them - if we can provide guidance on what the resulting URL will be, we should include that to make this easier.

Reply:
I'll leave this as a nit, but I'd still recommend we provide guidance on how to connect (if at all possible).